docs/design/multi-schedulers.md

```diff
@@ -0,0 +1,105 @@
+# Multi schedulers
+
+## Introduction
+
+Sometimes we may want to the cluster be divided into multi sections for different usage.
```

Comment:
As the feedback from the users, another scenario is to schedule workloads under ns `kube-system`  by default-scheduler and others by Volcano. So i suggest to take it into consideration, too.